src/Components/src/Microsoft.AspNetCore.Components/Microsoft.AspNetCore.Components.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Components feature for ASP.NET Core.</Description>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
This is because this PR adds a usage of `stackalloc`, and prior to `netstandard2.1`, that is always treated as unsafe.

Once we go to `netstandard2.1` the `unsafe` modifier can be removed.

Reply:
hmm, I think that's due to the language version not the TFM right? Either way I'm not too nervous about this as long as it works in wasm.